handbook/documentation.rst

```diff
@@ -0,0 +1,33 @@
+####
+Developer Documentation
+####
+
+Overview
+********
+
+Documentation can live in lot's of places, we want to be clear about which documentation lives where most of the time so that finding docs is easier.  In general, keep the documents as close to the thing that they are describing as possible.
+
+Details
+****
+
+* READMEs, HOW-TOs, API Docs and ADRs should live with the code.
+
+  * API docs should be in swagger format.
+
+  * **Everything else should be RSTs**.  Yes, you already know markdown but RST is more powerful and robust and that is what we have chosen to use for consistency across the whole platform.  This_ is a good resources for learning RST.
```

Comment:
* Maybe something like?:
```
* **Everything else should be RST format**.  `Learn about using RST`_.
```
* I think the sentence "`Yes, you...whole platform.`" should either be dropped, or maybe add something like "`Learn about why RST`" that could point to the right place in the OEP.  Another option is to provide a section of this doc with links to relevant decisions.

Reply:
Also, if RST is the format for everything in the platform, aren't README, HOWTOs, etc. living in the code also using RST? Ie recommendations about format might be broader than ones about location.